src/commands/force/data/soql/query.ts

```diff
@@ -182,8 +183,13 @@ export class DataSoqlQueryCommand extends DataCommand {
   public static readonly flagsConfig: FlagsConfig = {
     query: flags.string({
       char: 'q',
-      required: true,
```

Comment:
without this `sfdx force:data:soql:query -u <username>` will not throw an error if no query is passed, I don't think oclif provides a way to specify a set of exclusive **and** required flags, if neither `--query` or `--soqlqueryfile` is passed in the command should throw an error.

Reply:
oh nvm, we have `exactlyOne` available for this case:
https://github.com/salesforcecli/plugin-env/blob/3466e0ca76a317bafa3e754b75aa4fd00e1371b9/src/commands/env/create/scratch.ts#L53